problems/removeNumbersAtOddIndices.js

```diff
@@ -2,8 +2,15 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+const removeNumbersAtOddIndices = (arr) => {
+    let output = []
+    for(let i = 0; i < arr.length; i++){
+        if(i % 2 === 1){
+            output.push(arr[i])
+            arr = output
+        }
+    }
```

Comment:
Try using .filter for cleaner code. Remember the callback on array methods can also take the index as a second argument.